<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QLab_dbClass</class>
 <widget class="QMainWindow" name="QLab_dbClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QLab_db</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QTableView" name="tableView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>find student</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="nameRB">
         <property name="text">
          <string>name</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="surnameRB">
         <property name="text">
          <string>surname</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="phoneRB">
         <property name="text">
          <string>phone number</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textInput"/>
       </item>
       <item>
        <widget class="QPushButton" name="findBtn">
         <property name="text">
          <string>find</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="showAllBtn">
      <property name="text">
       <string>show all</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QLab_db.qrc"/>
 </resources>
 <connections/>
</ui>
